README.md

```diff
@@ -32,7 +32,7 @@ You can run `python3 --version` to check the version of Python installed.
 
 As a general practice, it is recommended to use a virtual environment for the installation.
 ```shell
-python3.9 -m venv virtenv
+python3.11 -m venv virtenv
```

Comment:
Can we keep this as 3.9, or is there a specific need for python 3.11?  We've been keeping python 3.9 as the minimum per PyPAM reqs (also 3.10 and 3.11).

Reply:
Yes, okay by me.  I do not think there is anything that requires 3.11.